src/Command/UpdateLicensesCommand.php

```diff
@@ -296,19 +296,25 @@ private function findAndCheckExtension(InputInterface $input, OutputInterface $o
         $output->writeln('');
     }
 
-    private function addLicenseToFile(SplFileInfo $file, string $startDelimiter = '\/', string $endDelimiter = '\/'): void
+    private function addLicenseToFile(SplFileInfo $file, string $startDelimiter = '\/', string $endDelimiter = '\/', string $startReplacer = null, string $endReplacer = null, ?string $commentDelimiter = null): void
```

Comment:
Can you provide details about the purpose of the new parameters `$startReplacer` and `$endReplacer`?
I understand we only need them for a one-time fix but I might be wrong.

Reply:
Yes it's only needed for the special use case to fix the twig files that have been modified by the new linter

Reply:
We could almost remove this after the 9.0.x branch has been fixed because it should be used only once, so it's debatable if it's worth integrating this in the tool